index.js

```diff
@@ -1097,6 +1097,20 @@ class Encore {
         return this;
     }
 
+    /**
+     * Enables image optimization with imagemin.
+     *
```

Comment:
Maybe add a note there that it will only be enabled for productions build by default + an example of how to use it with/without a callback.